index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,700;1,300;1,400&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" href="style.css">
+    <title>Jubaer Maruf</title>
+</head>
+<body>
+    <header class="header">
+        <nav class="nav">
+            <h2 class="logo">My Logo</h2>
+            <div class="hamburger">
+                <span class="span-burger burger-1"></span>
+                <span class="span-burger burger-2"></span>
+                <span class="span-burger burger-3"></span>
+            </div>
+        </nav>
+    </header>
+    <main class="main">
+        <section class="section-hero">
+            <img src="./images/full-circle.svg" alt="logo of circle" class="full-circle">
+            <img src="./images/stick.svg" alt="logo of stick" class="stick">
+            <img src="./images/half-circle.svg" alt="" class="bottom-half-circle">
+            <img src="./images/half-ring.svg" alt="logo of half ring" class="bottom-half-ring">
+            <img src="./images/ring.svg" alt="logo of ring" class="bottom-ring">
+            <img src="./images/full-circle.svg" alt="logo of circle" class="top-full-circle">
+            <img src="./images/stick.svg" alt="logo of stick" class="top-stick">
+            <img src="./images/half-circle.svg" alt="" class="top-half-circle">
+            <img src="./images/big-stick.svg" alt="logo of stick" class="top-left-stick">
+            <img src="./images/big-stick.svg" alt="logo" class="big-stick-right">
+            <img src="./images/big-ring.svg" alt="logo" class="big-ring-right">
```

Comment:
- [ ] You have done a great job to ensure that there is a background image appearing. However, you should know that this is for design purposes as we try to make our portfolio look nice. Therefore the background image should be added using `CSS` styles, not as an image ☺️.
Keynote: Kindly ensure to use one background image in CSS for easy formating.